react/components/ProgressBar/getOrderProgress.ts

```diff
@@ -40,34 +40,25 @@ const statusMap: StatusMap = {
   invoiced: FOURTH_STEP,
 }
 
-export default function getOrderProgress(status: string, packages: any) {
-  let progress = statusMap[status]
-  if (
-    progress === FOURTH_STEP &&
-    (isDelivered(packages) ||
-      isCarrierHandling(packages) ||
-      isOrderReadyToPickUp(packages))
-  ) {
-    progress = FIFTH_STEP
-  }
-  return progress
+function isCarrierHandling(packages: any) {
+  if (packages == null || packages.length === 0) return false
+
+  return packages.every(
+    (pck: any) => pck.package?.courierStatus?.data?.length > 0
```

Comment:
Curiosity here, does `pck.package` mean that the object is built as such?
```
packages: [
  package: {
    package: {courierStatus:{...}}
  }
]

Reply:
Yes, `packagify` (= `parcelify`) returns this object. See [this](https://github.com/vtex/delivery-packages#parcelifyorder-options)

> A Parcel object shape
> ```js
> {
>   address: Object,
>   listPrice: Number,
>   price: Number,
>   sellingPrice: Number,
>   pickupFriendlyName: String,
>   seller: String,
>   items: [Object],
>   package: Object,
>   selectedSla: String,
>   selectedSlaObj: Object,
>   slas: [Object],
>   shippingEstimate: String,
>   shippingEstimateDate: String,
>   deliveryChannel: String,
>   selectedSlaType: String,
>   deliveryIds: [Object]
> }
> ```